src/appleseed/renderer/kernel/rendering/debug/blanksamplerenderer.cpp

```diff
@@ -61,6 +62,11 @@ namespace
             delete this;
         }
 
+        void print_settings() const override
+        {
+        }
+
```

Comment:
Remove one blank line. Methods are separated by a single blank line (fix in other places too).